apps/onboarding/client/router.coffee

```diff
@@ -11,6 +11,8 @@ module.exports = class OnboardingRouter extends Backbone.Router
     'welcome/:scene': 'scene'
 
   initialize: ->
+    @route /welcome((\?)(.)+)?$/, 'welcome'
```

Comment:
?

Reply:
Catches any route with `?param=val` and sends it to the `welcome` route.

Reply:
I don't follow, why is this needed. Backbone router should just ignore the params and route to the correct path?

Reply:
I mean realistically if it's not loading it was just cuz of the router conflict caused by the fucking dueling routers with the blacklist not matching

Reply:
It does not ignore the params, they get set as a `:scene`

Reply:
https://github.com/jashkenas/backbone/issues/891

Reply:
That is insane.